### ISSUE TITLE
gPodder doesn't start

### ISSUE THREAD

**Issue by jeifour** [i-rinat/freshplayerplugin#216]
I get the following when trying to start gPodder.

```
[jei@GnosisNana ~]$ gpodder
1435177334.433182 [gpodder.sync] WARNING: Could not find eyed3.mp3
libva info: VA-API version 0.37.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib/dri/i965_drv_video.so
libva info: va_openDriver() returns -1
NOT SANDBOXED
libva info: VA-API version 0.37.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib/dri/i965_drv_video.so
libva info: va_openDriver() returns -1
[ERROR:flash/platform/pepper/common/pep_process.cpp(19)] CHECK(g_process_type == PROCESS_TYPE_UNKNOWN) failed
中止 (コアダンプ)
```
This is with freshplayerplugin 0.3.0-2 from AUR. This is output after uninstalling freshplayer:

```
[jei@GnosisNana ~]$ gpodder
1435177635.731032 [gpodder.sync] WARNING: Could not find eyed3.mp3
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.5) (Arch Linux build 7.u79_2.5.5-1-x86_64)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
```
It works fine then. Now I build freshplayer myself from git, output is:

```
[jei@GnosisNana ~]$ gpodder
1435177736.881789 [gpodder.sync] WARNING: Could not find eyed3.mp3
NOT SANDBOXED
[fresh] [warning] NP_Initialize was called more than once
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.5) (Arch Linux build 7.u79_2.5.5-1-x86_64)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
```
Also works now. So yes just wanted to let you know about this, seems this issue is fixed. But if gpodder starts not working again (it worked at some point in the past with freshplayerplugin installed) I will update this issue.

**Comment by i-rinat** [comment]
Yes, similar issue (also with Java and embedded browser engine) was fixed in 7243809dcbe8b2dac9695ee095367cc1ac2b40d4.

**Comment by i-rinat** [comment]
v0.3.1 was just released, so mentioned fix is already in.
